cgroup2/manager.go

```diff
@@ -242,8 +242,10 @@ func setResources(path string, resources *Resources) error {
 type CgroupType string
 
 const (
-	Domain   CgroupType = "domain"
-	Threaded CgroupType = "threaded"
+	Domain         CgroupType = "domain"
+	DomainThreaded CgroupType = "domain threaded"
+	DomainInvalid  CgroupType = "domain invalid"
+	Threaded       CgroupType = "threaded"
```

Comment:
@yaoyinnan @dcantah @mxpv  Wondering if additional changes were needed as part of this; I see `Manager.Delete()` has a special case for `Threaded`; should the same code be called when it encounters `DomainThreaded` ("domain threaded")?
https://github.com/containerd/cgroups/blob/d4e976de1b4e9f9bb4624a7f0860ab19e9aa995b/cgroup2/manager.go#L479-L490